lib/rubocop/cop/rails/freeze_time.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Identifies usages of `travel_to` with an argument of the current time and
+      # change them to use `freeze_time` instead.
+      #
+      # @safety
+      # This cop’s autocorrection is unsafe because `freeze_time` just delegates to
+      # `travel_to` with a default `Time.now`, it is not strictly equivalent to `Time.now`
+      # if the argument of `travel_to` is the current time considering time zone.
```

Comment:
Can you indent the safety description?

```suggestion
      # @safety
      #   This cop’s autocorrection is unsafe because `freeze_time` just delegates to
      #   `travel_to` with a default `Time.now`, it is not strictly equivalent to `Time.now`
      #   if the argument of `travel_to` is the current time considering time zone.
```

Reply:
Oops, I changed the indentation correctly.